client/src/Homepage.js

```diff
@@ -0,0 +1,116 @@
+
+    try {
+      const response = await fetch(`https://www.googleapis.com/books/v1/volumes?q=${title}+isbn:${isbn}&key=AIzaSyCQ7EBy8yMIYuOdh350WfDgEwWKGJSWAh4`)
+      console.log('This is the Modern books response', response)
+      const jsonData = await response.json();
+      if (response.ok) {
+        return console.log('My book requests', jsonData);
+        //placeholder until I have a more info page to render this information on a new page
+      }
+    }
+    catch (error) {
+      console.error('Error fetching image data', error)
+    }
+  }
+
+  const recommendationModern = [{ title: 'The+Overstory', isbn: '9780393356687', image: 'http://books.google.com/books/content?id=AmxFtwEACAAJ&printsec=frontcover&img=1&zoom=1&source=gbs_api' }, { title: 'Dune', isbn: '9780143111580', image: 'http://books.google.com/books/content?id=ydQiDQAAQBAJ&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }, { title: 'Gone+Girl', isbn: '9780553418361', image: 'http://books.google.com/books/content?id=pd6MDQAAQBAJ&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }, { title: 'The+Circle', isbn: '9780385351409', image: 'http://books.google.com/books/content?id=sbxWAAAAQBAJ&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }, {
+    title: 'The+Goldfinch', isbn: '9780316248679', image: 'http://books.google.com/books/content?id=dvuK7isszLIC&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api'}, { title: 'Piranesi', isbn: '9781635575644', image: 'http://books.google.com/books/content?id=FCTYDwAAQBAJ&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }, { title: 'Fun+Home', isbn: '9780618871711', image: 'http://books.google.com/books/content?id=eq0n9Ck79ysC&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }, { title: 'Pachinko+(National+Book+Award+Finalist)', isbn: '9781455563913', image: 'http://books.google.com/books/content?id=cxteDAAAQBAJ&printsec=frontcover&img=1&zoom=1&edge=curl&source=gbs_api' }]
+
+  const mapModern = recommendationModern.map(recommendationModern =>
+    <img className='p-2 img-fluid recommended-books' onClick={() => handleImageClick(recommendationModern.title, recommendationModern.isbn)} key={recommendationModern.isbn} alt={recommendationModern.title} src={recommendationModern.image} />
+  )
+
+  return (
+    <div className='pt-5 px-3'>
+      <div>
+        <h3>Modern Classics</h3>
+      </div>
+      <div style={{ backgroundColor: '#D9D9D9'}}>
+        <div className='px-3 row justify-content-between'>
+          {mapModern}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function RecommendationThisMonth() {
+
+
+  const handleImageClick = async (title, isbn) => {
+
+    try {
+      const response = await fetch(`https://www.googleapis.com/books/v1/volumes?q=${title}+isbn:${isbn}&key=AIzaSyCQ7EBy8yMIYuOdh350WfDgEwWKGJSWAh4`)
+      console.log(response)
+      const jsonData = await response.json();
+      if (response.ok) {
```

Comment:
And here.

Since you have several places where you call fetch and handle the response, it might be worthwhile to extract all those fetch calls into standalone functions. This would also mean you only have to put the key in one place.